### PR TITLE
CNV#40180: Release note: API is now v1beta1

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.17

Issue: [CNV-40180](https://issues.redhat.com/browse/CNV-40180)

Link to docs preview: See the note in the Storage section of the 4.17 OpenShift Virtualization release notes. (https://81176--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-17-release-notes.html#virt-4-17-storage)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@mhenriks, can you please review this release note, which corresponds to the feature [CNV- 40179](https://issues.redhat.com/browse/CNV-40179)? Thanks.

**Review by jpeimer — APPROVED** [review]

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 1]
```diff
@@ -72,6 +72,9 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-17-storage"]
 === Storage
 
+//CNV-40180 Release note: API is now v1beta1
+* The VirtualMachineSnapshot API version is now v1beta1.
```
Comment (lahinson) [inline virt/release_notes/virt-4-17-release-notes.adoc:76]:
```suggestion
* The `VirtualMachineSnapshot` API version is now v1beta1.
```
If the name of this API version is not typically in monospace font, feel free to ignore this suggestion.

**Review by lahinson — COMMENTED** [review]
Just one suggestion. Otherwise, LGTM.

File: virt/release_notes/virt-4-17-release-notes.adoc (hunk 1 above)
Comment (aspauldi) [reply virt/release_notes/virt-4-17-release-notes.adoc]: It is! Good catch; thanks!

**Pull request merged**